package.json

```diff
@@ -241,10 +241,10 @@
   "dependencies": {
     "@elastic/datemath": "^5.0.3",
     "@elastic/eui": "34.6.0",
-    "@redis-ui/components": "^38.1.4",
-    "@redis-ui/icons": "^4.16.1",
-    "@redis-ui/styles": "^11.4.0",
-    "@redis-ui/table": "^2.4.0",
+    "@redis-ui/components": "^39.18.0",
+    "@redis-ui/icons": "^6.1.1",
+    "@redis-ui/styles": "^12.9.0",
```

Comment:
There is something wrong with the colors on our end, using dark theme and a [notice banner](https://redislabsdev.github.io/redis-ui/?path=/story/components-banner--semantic-banners), in the Redis UI Storybook, it's purple, but in Redis Insight, it still looks pinkish.

The same thing is valid for the toast notifications that use the same variant.

<img width="3326" height="1880" alt="Image" src="https://github.com/user-attachments/assets/b026f6ce-1fd3-4110-a87e-eb65a633fd07" />

<img width="3764" height="1886" alt="Image" src="https://github.com/user-attachments/assets/9532de60-ec1c-4d1c-b7ec-367a8d23285f" />